SCNRecorder.podspec

```diff
@@ -1,12 +1,12 @@
 Pod::Spec.new do |s|
   s.name                  = 'SCNRecorder'
-  s.version               = '2.8.0'
+  s.version               = '2.8.1'
   s.summary               = 'A lags-free recorder of ARKit and SceneKit for iOS in Swift'
   s.homepage              = 'https://github.com/gorastudio/SCNRecorder'
   s.license               = { :type => 'MIT', :file => 'LICENSE.md' }
   s.author                = { 'Vladislav Grigoryev' => 'dev.grigoriev@gmail.com' }
   s.platform              = :ios, '12.0'
-  s.source                = { :git => 'https://github.com/gorastudio-ceo/SCNRecorder.git', :tag => s.version.to_s }
+  s.source                = { :git => 'https://github.com/amir-cg/SCNRecorder.git', :tag => s.version.to_s }
```

Comment:
You have to revert this, otherwise pod will use your repo :)

Reply:
done